apps/dashboard/src/utils/code-snippets.ts

```diff
@@ -0,0 +1,193 @@
+import { API_HOSTNAME } from '@/config';
```

Comment:
this file contains snippets for different languages; it might seem to you that the code is wrong-formatted, but it's not, it renders properly in the editor

Reply:
btw it's copied from the old Web app but slightly improved interface